shared/wallets/wallet/settings/index.js

```diff
@@ -0,0 +1,186 @@
+// @flow
+import * as React from 'react'
+import * as I from 'immutable'
+import * as Kb from '../../../common-adapters'
+import * as Styles from '../../../styles'
+import * as Types from '../../../constants/types/wallets'
+
+export type SettingsProps = {
+  accountID: Types.AccountID,
+  name: string,
+  user: string,
+  isDefault: boolean,
+  currency: Types.Currency,
+  currencies: I.List<Types.Currency>,
+  onBack: () => void,
+  onDelete: () => void,
+  onSetDefault: () => void,
+  onEditName: () => void,
+  onCurrencyChange: (currency: Types.CurrencyCode) => void,
+  refresh: () => void,
+}
+
+const makeDropdownItems = (currencies: I.List<Types.Currency>, currency: Types.Currency) => {
+  const items = [
+    <Kb.Box2 centerChildren={true} direction="vertical" key="_header">
+      <Kb.Text type="BodySmall" style={styles.dropdownHeader}>
+        Past transactions won't be affected by this change.
+      </Kb.Text>
+    </Kb.Box2>,
+  ]
+  // spread the List into an array with [...]
+  return items.concat([...currencies].map(s => makeDropdownItem(s, s.code === currency.code)))
+}
+
+const makeDropdownItem = (item: Types.Currency, isSelected: boolean) => (
+  <Kb.Box2 centerChildren={true} direction="vertical" fullWidth={true} key={item.code}>
+    <Kb.Text
+      type="Header"
+      style={Styles.collapseStyles([styles.centerText, isSelected && styles.itemSelected])}
+    >
+      {item.description}
+    </Kb.Text>
+  </Kb.Box2>
+)
+
+const AccountSettings = (props: SettingsProps) => {
```

Comment:
Renamed `SettingsPopup` to `AccountSettings` so we can keep the `popup` terminology for `MaybePopup` components.